qml/pages/OverviewPage.qml

```diff
@@ -368,6 +384,21 @@ Page {
                 size: BusyIndicatorSize.Large
             }
         }
+
```

Comment:
Perhaps you could use/change the existing loadingLabel (with a different text, obviously) to reduce the amount of qml items?

Reply:
Thanks for the suggestion @jgibbon, I have merged the two Busy Indicators to one now.